gcloud: add German translation (#10961)
tldr-pages/tldr: pages.de/common/gcloud.md

# gcloud

> Das offizielle CLI-Tool für die Google Cloud Platform.
> Weitere Informationen: https://cloud.google.com/sdk/gcloud.

> - Alle Eigenschaften in deiner aktiven Konfiguration auflisten:

Reviewer: ```suggestion
- Liste alle Eigenschaften der aktiven Konfiguration auf:
```

`gcloud config list`

> - Bei deinem Google-Konto anmelden:

Reviewer: ```suggestion
- Mit einem Google-Konto anmelden:
```

Imperative is very hard in this case, I wouldn't know how to do it.

`gcloud auth login`

> - Das aktive Projekt festlegen:

Reviewer: ```suggestion
- Lege das aktive Projekt fest:
```

`gcloud config set project {{Projektname}}`

> - Eine SSH-Verbindung zu einer virtuellen Maschineninstanz herstellen:

Reviewer: ```suggestion
- Stelle eine SSH-Verbindung zu einer virtuellen Maschineninstanz her:
```

`gcloud compute ssh {{Benutzer}}@{{Instanz}}`

> - Alle Google Compute Engine-Instanzen in einem Projekt anzeigen. Instanzen aus allen Zonen werden standardmäßig aufgelistet:

Reviewer: ```suggestion
- Zeige alle Google Compute Engine-Instanzen in einem Projekt an. Standardmäßig werden Instanzen aus allen Zonen aufgelistet:
```

`gcloud compute instances list`

> - Eine kubeconfig-Datei mit den entsprechenden Anmeldeinformationen aktualisieren, um kubectl auf einen bestimmten Cluster in Google Kubernetes Engine auszurichten:

Reviewer: ```suggestion
- Aktualisiere eine kubeconfig-Datei mit den entsprechenden Anmeldeinformationen, um kubectl auf einen bestimmten Cluster in Google Kubernetes Engine auszurichten:
```

`gcloud container clusters get-credentials {{Clustername}}`

> - Alle gcloud CLI-Komponenten aktualisieren:

Reviewer: ```suggestion
- Aktualisiere alle gcloud CLI-Komponenten:
```

`gcloud components update`

> - Hilfe für einen bestimmten Befehl anzeigen:

Reviewer: ```suggestion
- Zeige Hilfe für einen bestimmten Befehl an:
```

`gcloud help {{Befehl}}`

> 

Reviewer: ```suggestion
```

Reviewer: Remove empty line
